### DISCUSSION TITLE
new_chat_member first_name

### DISCUSSION THREAD

**Discussion by fossildj** [nutgram/nutgram#423] [Q&A] [answered]
Hi, i am using the onNewChatMembers

$bot->onNewChatMembers(function (Nutgram $bot){
    $new_member = $bot->update()->message->new_chat_members->first_name;

However, i could not get the first_name of the new member

**Comment by Lukasss93** [comment, marked as answer]
new_chat_members is an array...

https://github.com/nutgram/nutgram/blob/fe4fd1a44f4a8214f0e6f7cb645ecdf42f42e3c9/src/Telegram/Types/Message/Message.php#L270
